apps/schools/domains/query/components/queryList/index.tsx

```diff
@@ -47,29 +51,19 @@ export function QueryList() {
             text: obj.text,
             color: obj.color,
             count: analytics ? (analytics.analytics as unknown as { [index: string]: number })[key] : 0,
-            isSelected: selectedStatuses.has(key),
+            isSelected: statuses?.includes(key) ?? false,
             onClick: () => {
-                selectedStatuses.add(key)
-                setStatusFilter(Array.from(selectedStatuses).join(','))
+                setStatuses(x => [...(x ?? []), key])
             },
             onExit: () => {
-                setStatusFilter(
-                    Array.from(selectedStatuses)
-                        .filter((value) => value != key)
-                        .join(','),
-                )
+                setStatuses(x => {
+                    const res = [...(x ?? []).filter(y => y != key)];
+                    return res.length === 0 ? null : res;
+                })
```

Comment:
use understandable variable names

Reply:
)

Reply:
It is understandable, isn't it? I set statuses so delegate inside has status type. I think it is not fundamental stuff.